tachyon/zk/plonk/expressions/evaluator/selector_replacer_unittest.cc

```diff
@@ -1,20 +1,22 @@
-#include "tachyon/zk/expressions/evaluator/selector_replacer.h"
+#include "tachyon/zk/plonk/expressions/evaluator/selector_replacer.h"
 
 #include <memory>
 
-#include "tachyon/zk/expressions/evaluator/test/evaluator_test.h"
-#include "tachyon/zk/expressions/expression_factory.h"
+#include "tachyon/zk/plonk/expressions/evaluator/simple_selector_finder.h"
+#include "tachyon/zk/plonk/expressions/evaluator/test/evaluator_test.h"
+#include "tachyon/zk/plonk/expressions/expression_factory.h"
 
-namespace tachyon::zk::evaluator {
+namespace tachyon::zk::plonk {
 
-using Expr = std::unique_ptr<Expression<GF7>>;
-
-class SelectorReplacerTest : public EvaluatorTest {};
+class SelectorReplacerTest : public EvaluatorTest {
+ public:
+  using Expr = std::unique_ptr<Expression<GF7>>;
+};
 
 TEST_F(SelectorReplacerTest, Constant) {
   GF7 value = GF7::Random();
   Expr expr = ExpressionFactory<GF7>::Constant(value);
-  EXPECT_EQ(*expr, *ReplaceSelectors(expr.get(), {}, false));
+  EXPECT_EQ(*expr, *ReplaceSelectors<GF7>(expr.get(), {}, false));
```

Comment:
Please remove `GF7`, since it can be inferenced.

```suggestion
  EXPECT_EQ(*expr, *ReplaceSelectors(expr.get(), {}, false));
```